src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,17 @@
+package racingcar.domain
+
+
+class Car(
+    val position: Position = Position(POSITION_FIRST),
+) {
+
+    fun moveCar(): Car {
```

Comment:
```suggestion
    fun move(): Car {
```
클래스명만 보아도, Car와 관련된 책임을 갖는 클래스이기 때문에 Car를 작성하지 않으셔도 됩니다. 😅